inc/class-wpseo-meta.php

```diff
@@ -634,20 +634,20 @@ public static function remove_meta_if_default( $null, $object_id, $meta_key, $me
 	 *
 	 * @static
 	 *
-	 * @param  null   $null       Old, disregard.
+	 * @param  null   $check      The current status to allow adding metadata for the given type
```

Comment:
Could you change the variable type to `null|bool`.
Could you add a period `.` at the end of the line.